metabase/metabase
# Make FileInput assessible

Epic: https://github.com/metabase/metabase/issues/19696

Changes:
- Adds keyboard navigation
- Adds focus ring

How to test:
- Go to Admin > Databases > Add database
- Try to add a new PostgreSQL database with `SSL` and `Authenticate client certificate` enabled. Use `Uploaded path` as the option and try to upload a file.
- Make sure that file contents are properly send to the backend. You do not need to check that the database has been properly created.
